### PR TITLE
PREAPPS:800:: Add SMIME Prefs

### PR DESCRIPTION
**Pull request by SantoshKShetty**
- Pull zimbraFeatureSMIMEEnabled from backend

### CODE REVIEW

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ import {
 	FolderView,
 	InviteReplyInput,
 	PreferencesInput,
+	PropertiesInput,
```
Comment (silentsakky) [inline src/batch-client/index.ts]: remove this as we are using metadata to save smime operation

File: src/batch-client/index.ts [hunk 2]
```diff
@@ -460,6 +461,15 @@ export class ZimbraBatchClient {
 			}
 		});
 
+	public modifyZimletProperties = (props: PropertiesInput) =>
```
Comment (silentsakky) [inline src/batch-client/index.ts]: same as above

File: src/schema/schema.graphql [hunk 3]
```diff
@@ -776,6 +776,7 @@ type AccountInfo {
 	signatures: Signatures
 	attrs: AccountInfoAttrs
 	prefs: Preferences
+	zimlets: Zimlets
```
Comment (silentsakky) [inline src/schema/schema.graphql]: same as above

File: src/schema/schema.graphql [hunk 4]
```diff
@@ -886,6 +888,28 @@ type Preferences {
 	zimbraPrefShowFragments: Boolean
 }
 
+type ZimletUserProperty {
```
Comment (silentsakky) [inline src/schema/schema.graphql]: same as above

File: src/schema/schema.graphql [hunk 5]
```diff
@@ -1313,6 +1339,11 @@ input PreferencesInput {
 	zimbraPrefShowFragments: Boolean
 }
 
+input PropertiesInput {
```
Comment (silentsakky) [inline src/schema/schema.graphql]: same as above

File: src/schema/schema.graphql [hunk 6]
```diff
@@ -1750,6 +1781,7 @@ type Mutation {
 	modifyAppointment(accountName: String, appointment: CalendarItemInput!): Boolean
 	modifyIdentity(id: ID!, attrs: IdentityAttrsInput!): String
 	modifyPrefs(prefs: PreferencesInput!): Boolean
+	modifyZimletProperties(props: PropertiesInput!): Boolean
```
Comment (silentsakky) [inline src/schema/schema.graphql]: same as above

File: src/schema/schema.ts [hunk 7]
```diff
@@ -9,6 +9,7 @@ import {
 	InviteReplyInput,
 	NameIdInput,
 	PreferencesInput,
+	PropertiesInput,
```
Comment (silentsakky) [inline src/schema/schema.ts]: same as above

File: src/schema/schema.ts [hunk 8]
```diff
@@ -261,6 +262,8 @@ export function createZimbraSchema(
 					zimbra.account.modifyIdentity(id, attrs),
 				modifyPrefs: (_, { prefs }) =>
 					client.modifyPrefs(prefs as PreferencesInput),
+				modifyZimletProperties: (_, { props }) =>
```
Comment (silentsakky) [inline src/schema/schema.ts]: same as above

File: src/batch-client/index.ts (hunk 1 above)
Comment (SantoshKShetty) [reply src/batch-client/index.ts]: Done

File: src/batch-client/index.ts (hunk 2 above)
Comment (SantoshKShetty) [reply src/batch-client/index.ts]: Done

File: src/schema/schema.graphql (hunk 3 above)
Comment (SantoshKShetty) [reply src/schema/schema.graphql]: Done

File: src/schema/schema.graphql (hunk 4 above)
Comment (SantoshKShetty) [reply src/schema/schema.graphql]: Done

File: src/schema/schema.graphql (hunk 5 above)
Comment (SantoshKShetty) [reply src/schema/schema.graphql]: Done

File: src/schema/schema.graphql (hunk 6 above)
Comment (SantoshKShetty) [reply src/schema/schema.graphql]: Done

File: src/schema/schema.ts (hunk 7 above)
Comment (SantoshKShetty) [reply src/schema/schema.ts]: Done

File: src/schema/schema.ts (hunk 8 above)
Comment (SantoshKShetty) [reply src/schema/schema.ts]: Done

**Pull request merged**